webapp/src/ts/services/target-aggregates.service.ts

```diff
@@ -37,6 +36,23 @@ export class TargetAggregatesService {
     private ngZone:NgZone,
   ) { }
 
+  private readonly NBR_MONTHS = 3;
```

Comment:
What is NBR?

Reply:
short for NUMBER :)

Reply:
I didn't think about `number`. What about `TARGET_MAX_MONTHS_OLD` or `MAX_MONTHS_OLD`